terraform/deployments/tfc-aws-config/gcp_oidc.tf

```diff
@@ -0,0 +1,63 @@
+data "google_project" "project" {}
+
+resource "google_iam_workload_identity_pool" "tfc" {
+  workload_identity_pool_id = "terraform-cloud-${var.govuk_environment}"
+  display_name              = "Terraform Cloud (${var.govuk_environment})"
+  description               = "Identity pool for Terraform Cloud connection to GCP (${var.govuk_environment})"
+}
+
+resource "google_iam_workload_identity_pool_provider" "tfc" {
+  workload_identity_pool_id          = google_iam_workload_identity_pool.tfc.workload_identity_pool_id
+  workload_identity_pool_provider_id = "terraform-cloud-${var.govuk_environment}"
+  attribute_condition                = "assertion.sub.startsWith(\"organization:govuk:\")"
```

Comment:
Do we also need to assert on audience (`aud`) or is this sufficient? (apologies if this is a poorly-formed question!)

Reply:
Ah cool, sounds like aud isn't very useful here then! Thanks!